python/ray/_private/utils.py

```diff
@@ -1899,3 +1899,19 @@ def try_import_each_module(module_names_to_import: List[str]) -> None:
             importlib.import_module(module_to_preload)
         except ImportError:
             logger.exception(f'Failed to preload the module "{module_to_preload}"')
+
+
+def update_envs(env_vars: Dict[str, str]):
+    if env_vars is None:
+        return
+    env_keys = [
+        "PATH",
+        "LD_LIBRARY_PATH",
+        "DYLD_LIBRARY_PATH",
+        "LD_PRELOAD",
+    ]
+    for key, value in env_vars.items():
+        if key in env_keys and key in os.environ:
```

Comment:
I think this will break backwards compatibility.

Reply:
@cadedaniel  In fact, there will be no compatibility issues. I deliberately put the custom env of the runtime env in front.

Reply:
Seems that today users can overwrite `PATH` et al with runtime envs. With this change they will only be able to append to it. Do I understand correctly?

Reply:
For the scenes you reminded, I have changed it to the above form, allowing users to choose whether to overwrite or append.
![image](https://user-images.githubusercontent.com/11072802/232666705-029016b2-d73e-4017-ba95-ae4e129bc25a.png)